packages/access-api/src/routes/validate-email.js

```diff
@@ -134,6 +139,93 @@ async function session(req, env) {
     req.query.ucan,
     delegation.capabilities[0].nb.key
   )
+  if (req.method.toLowerCase() === 'post') {
+    const accessSessionResult = await validator.access(delegation, {
+      capability: Access.session,
+      principal: Verifier,
+      authority: env.signer,
+    })
+    if (accessSessionResult.error) {
+      throw new Error(
+        `unable to validate access session: ${accessSessionResult.error}`
+      )
+    }
+    const accountDID = accessSessionResult.audience.did()
+    // @todo: use new ucanto `Account` instead
+    const accountOneOffKey = await ed25519.generate()
+    const account = accountOneOffKey.withDID(accountDID)
+
+    const agentPubkey = accessSessionResult.capability.nb.key
+
+    const update = await ucanto.delegate({
+      issuer: env.signer,
+      audience: account,
+      capabilities: [
+        {
+          with: env.signer.did(),
+          can: './update',
+          nb: {
+            key: agentPubkey,
+          },
+        },
+      ],
+    })
+    const attestKeyAuthorizesAccount = await ucanto.delegate({
+      issuer: env.signer,
+      audience: { did: () => agentPubkey },
+      capabilities: [
+        {
+          with: env.signer.did(),
+          can: 'ucan/attest',
+          nb: {
+            proof: update.cid,
+          },
+        },
+      ],
+      proofs: [update],
+    })
+
+    // create delegations that should be claimable
+    const delegationAccountToKey = await ucanto.delegate({
```

Comment:
This is somewhat correct, except it will have a signature from `accountOneOffKey` key as opposed to having an "absent" signature. Which may be ok temporarily, but not in MVP because those could fail to validate.